servicetalk-http-api/src/testFixtures/java/io/servicetalk/http/api/TestStrategyInfluencerAwareConversions.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2019 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.http.api;
+
+import java.util.function.Predicate;
+
+public final class TestStrategyInfluencerAwareConversions {
+
+    private TestStrategyInfluencerAwareConversions() {
+    }
+
+    public static StreamingHttpClientFilterFactory toConditionalClientFilterFactory(
+            final Predicate<StreamingHttpRequest> predicate, final StreamingHttpClientFilterFactory original) {
+        return StrategyInfluencerAwareConversions.toConditionalClientFilterFactory(predicate, original);
+    }
+
+    public static StreamingHttpConnectionFilterFactory toConditionalConnectionFilterFactory(
+            final Predicate<StreamingHttpRequest> predicate, final StreamingHttpConnectionFilterFactory original) {
+        return StrategyInfluencerAwareConversions.toConditionalConnectionFilterFactory(predicate, original);
```

Comment:
nit: static import ?

Reply:
Yes, it clashes. 

Should I better rename `TestStrategyInfluencerAwareConversions#toConditionalClientFilterFactory`, `TestStrategyInfluencerAwareConversions.toConditionalConnectionFilterFactory` to something else to use static import?

Reply:
We do not have to go out of our way to use static imports. I would suggest prefering intuitive names over the intent to use static imports.

Reply:
Done! Now we have `TestStrategyInfluencerAwareConversions.conditionalConnectionFilterFactory`
`TestStrategyInfluencerAwareConversions.conditionalClientFilterFactory`